docs/firmware/payload_uboot.md

```diff
@@ -0,0 +1,32 @@
+U-Boot as a payload to OpenSBI
+==============================
+
+U-Boot is an open-source primary boot loader. It can be used as first and/or
+second stage boot loader in an embedded environment. Here, it is used as next
+stage bootloader to OpenSBI if given as a payload to OpenSBI.
+
+Currently, some of the required patches are not mainline. Thus, you need to
+cherry-pick the following out-of-tree patch series until it is accepted in mainline.
```

Comment:
The current stable upstream code of U-boot does not yet include all patches necessary to fully support OpenSBI. To use U-Boot as an OpenSBI payload, the following out-of-tree patch series must be applied to the upstream U-Boot source code.